Doc/library/glob.rst

```diff
@@ -49,8 +49,9 @@ For example, ``'[?]'`` matches the character ``'?'``.
       single: **; in glob-style wildcards
 
    If *recursive* is true, the pattern "``**``" will match any files and zero or
-   more directories and subdirectories.  If the pattern is followed by an
-   ``os.sep``, only directories and subdirectories match.
+   more directories, subdirectories and symbolic links to directories. If the
+   pattern is followed by an ``os.sep`` or ``os.altsep`` then files will not
```

Comment:
Adding Sphinx roles will turn the text into hyperlinks ([os.sep](https://docs.python.org/3.9/library/os.html?highlight=os#os.sep) and [os.altsep](https://docs.python.org/3.9/library/os.html?highlight=os#os.altsep)), so that readers can easily click on them to find the documentation for each of the data values.

```suggestion
   pattern is followed by an :data:`os.sep` or :data:`os.altsep` then files will not
```

Reply:
Good opportunity to piggy-back these, will do. If I use github's web interface to accept and commit this change, would the approver eventually "squash and merge" the extra commit?

Reply:
Yep, all of the commits made to the PR branch are squshed into a single commit. This happens regardless of whether you push it locally from the command line or GitHub's web interface. The GitHub web interface just makes it more convenient to add minor suggestions. (: